processing/data_collection/gazette/pipelines.py

```diff
@@ -22,6 +20,30 @@ def process_item(self, item, spider):
         return item
 
 
+class ExtraIssuePipeline:
```

Comment:
Please, remove this pipeline item. The file data processing is under a change. We will move the processing pipeline for another repository. It's the same that I sent you in another PR.

Reply:
For now, if you can identify the type from the web page, url or file name would be nice.

Reply:
@jvanz That's the only way I found to identify it. There's no information on the page, URL or filename. Do you still want me to remove it?

Reply:
@rennerocha @ejulio What do you think?

Reply:
If we're moving the processing out of this repository, I guess there isn't much to be done.
For sure we can keep the flag in the item, to avoid running the regex later.

Reply:
Do you mean keep the flag as `False`?

Reply:
Yes, if you do not have other way to identify it. Keep the value false.

Reply:
Done.